frontend/packages/dev-console/src/components/monitoring/metrics/MetricsDropdown.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import * as fuzzy from 'fuzzysearch';
+import { RootState } from '@console/internal/redux';
+import { Dropdown } from '@console/internal/components/utils';
+import { queryBrowserRunQueries, queryBrowserPatchQuery } from '@console/internal/actions/ui';
+import { getActiveNamespace } from '@console/internal/reducers/ui';
+import { QueryObj } from '@console/internal/components/monitoring/query-browser';
+import { metricsQuery, getTopMetricsQueries } from '../queries';
+import './MetricsDropdown.scss';
+
+const ADD_NEW_QUERY = '#ADD_NEW_QUERY#';
+
+interface StateProps {
```

Comment:
I see a mix between type and interface. Choose one and stick to it in the same file.

I don't think there is an agreement on preference overall but at least in the same file we should be consistent.

Reply:
have made use of type across